### PR TITLE
docs/running-on-mesos.md: state default values in default column

### PR DESCRIPTION
**Pull request by felixb**
This PR simply uses the default value column for defaults.

### CODE REVIEW

File: docs/running-on-mesos.md [hunk 1]
```diff
@@ -332,21 +332,21 @@ See the [configuration page](configuration.html) for information on Spark config
...
-  <td>Framework secret to authenticate to Mesos</td>
+  <td>(none)/td>
   <td>
     Set the secret with which Spark framework will use to authenticate with Mesos.
   </td>
 </tr>
 <tr>
   <td><code>spark.mesos.role</code></td>
-  <td>Role for the Spark framework</td>
+  <td>*</td>
```
Comment (rxin) [inline docs/running-on-mesos.md]: can you put <code> around this?

**Comment by rxin** [comment]
Actually I will just merge this and address the comment when I merge.

File: docs/running-on-mesos.md (hunk 1 above)
Comment (felixb) [inline docs/running-on-mesos.md]: I don't see anything.
Should I set it to `"*"`, `(*)` or ` * ` with blanks on each side?

Comment (rxin) [inline docs/running-on-mesos.md]: Oh I meant 
```
<code>*</code>
```

Comment (rxin) [inline docs/running-on-mesos.md]: I've already merged this though so don't worry about it.

**Pull request closed without merging**